src/python/grpcio/commands.py

```diff
@@ -264,6 +264,41 @@ def run(self):
       self.distribution.fetch_build_eggs(self.distribution.tests_require)
 
 
+class TestLite(setuptools.Command):
+  """Command to run tests without fetching or building anything."""
+
+  description = 'run tests without fetching or building anything.'
+  user_options = []
+
+  def initialize_options(self):
+    pass
+
+  def finalize_options(self):
+    # distutils requires this override.
+    pass
+
+  def run(self):
+    self._add_eggs_to_path()
+
+    import tests
+    loader = tests.Loader()
+    loader.loadTestsFromNames(['tests'])
+    runner = tests.Runner()
+    result = runner.run(loader.suite)
+    if not result.wasSuccessful():
+      sys.exit(1)
```

Comment:
Could we change this to `sys.exit('Test failure')`? If we need to make it any more detailed we can do so, but right now I'd rather like to see Python provide, independent of the runner behavior, a little bit more than just an error code on exit.